### PR TITLE
add attachments handling

### PR DESCRIPTION
**Pull request by ehborisov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -128,6 +128,11 @@
             <artifactId>jackson-module-jaxb-annotations</artifactId>
             <version>2.7.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-core</artifactId>
+            <version>1.7</version>
```
Comment (baev) [inline pom.xml]: 1.14

**Review by baev — CHANGES_REQUESTED** [review]

File: src/main/java/io/qameta/allure/AllureConstants.java [hunk 2]
```diff
@@ -18,6 +21,12 @@
 
     public static final String TEST_RUN_JSON_FILE_GLOB = "*-testrun.json";
 
+    public static final String ATTACHMENT_FILE_SUFFIX = "-attachment";
+
+    public static final Charset ATTACHMENT_ENCODING = StandardCharsets.UTF_8;
+
+    public static final Integer MAX_TITLE_LENGTH = 120;
```
Comment (baev) [inline src/main/java/io/qameta/allure/AllureConstants.java]: unused?

**Review by baev — CHANGES_REQUESTED** [review]

File: src/main/java/io/qameta/allure/AllureConstants.java (hunk 2 above)
Comment (ehborisov) [reply src/main/java/io/qameta/allure/AllureConstants.java]: it's used externally from aspects, should I move it there?

Comment (baev) [reply src/main/java/io/qameta/allure/AllureConstants.java]: As I sad before we don't need to limit titles anymore, so we need to remove this and all related functionality

Comment (ehborisov) [reply src/main/java/io/qameta/allure/AllureConstants.java]: my bad, all done.

File: src/main/java/io/qameta/allure/AllureConstants.java [hunk 3]
```diff
@@ -18,6 +21,10 @@
 
     public static final String TEST_RUN_JSON_FILE_GLOB = "*-testrun.json";
 
+    public static final String ATTACHMENT_FILE_SUFFIX = "-attachment";
+
+    public static final Charset ATTACHMENT_ENCODING = StandardCharsets.UTF_8;
```
Comment (baev) [inline src/main/java/io/qameta/allure/AllureConstants.java]: lets remove this as well

**Review by baev — CHANGES_REQUESTED** [review]

**Review by baev — APPROVED** [review]

**Comment by baev** [comment]
@ehborisov thanks!

**Pull request merged**